packages/web-react/jest.config.ts

```diff
@@ -0,0 +1,9 @@
+const config = {
+  preset: 'jest-config-spirit/jsdom',
+
+  // A list of paths to modules that run some code to configure or set up the testing framework before each test file in the suite is executed.
+  // https://jestjs.io/docs/configuration#setupfilesafterenv-array
+  setupFilesAfterEnv: ['<rootDir>/config/jest/setupTestingLibrary.ts'],
```

Comment:
Note to myself: Add a comment about that this should be moved to the `jest-config-spirit`.

Reply:
Comment added.